src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -26,6 +26,13 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 
 	try {
 		const url = new URL(`${req.protocol}://${req.get("host")}${req.originalUrl}`);
+		if (githubToken && !servers.hasCloudServer && !servers.gheServerInfos.length) {
```

Comment:
Looks like this conditional and the one below are checking if there's a GH token and if there is/isn't servers connected. What if there is no GH token? What if there's a token and the customer is only connected to cloud? This means they would see this screen.

I thought we were rendering this page only if there was no GH token otherwise:
- render the form for customers that have cloud/server connected
- render the select a product screen for customers that have cloud + server connected.

Reply:
Condition updated to check for the `gitHubToken` OR the number of connections.